<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
    background-color: rgb(128, 128, 128);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>671</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="indicator3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
 max-width:20px;
 max-height:20px;
 min-width:20px;
 min-height:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="indicator1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
 max-width:20px;
 max-height:20px;
 min-width:20px;
 min-height:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="indicator2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
 max-width:20px;
 max-height:20px;
 min-width:20px;
 min-height:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="indicator4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
 max-width:20px;
 max-height:20px;
 min-width:20px;
 min-height:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="indicator5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
 max-width:20px;
 max-height:20px;
 min-width:20px;
 min-height:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>470</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7f8185;
image: url(&quot;:/power_button.jpg&quot;);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="deliverShock">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7f8185;
image: url(&quot;:/deliver_shock.jpg&quot;);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="adultChild">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>490</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7f8185;
image: url(&quot;:/adult_child.jpg&quot;);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selfTest">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7f8185;
image: url(&quot;:/self_test_not_ok.jpg&quot;);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>287</width>
      <height>398</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>18</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Health State</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="health">
       <item>
        <property name="text">
         <string>V_TACH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>V_FIB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ASYSTOLE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PEA</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="age">
       <item>
        <property name="text">
         <string>Adult</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Child</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Infant</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Connect Electrode to AED</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="connectAED">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Place Pads on the Chest</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="connectChest">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Battery</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QProgressBar" name="battery">
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="decrease">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #4169e1;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="increase">
       <property name="styleSheet">
        <string notr="true">background-color: #4169e1;</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Compression Depth(cm)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="depth">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>7.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="confirm_patient">
       <property name="styleSheet">
        <string notr="true">background-color: #4169e1;</string>
       </property>
       <property name="text">
        <string>Confirm Patient</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="charge_battery">
       <property name="styleSheet">
        <string notr="true">background-color: #4169e1;</string>
       </property>
       <property name="text">
        <string>Charge Battery</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="textDisplay">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>160</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-right: none;
background:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="elapsedtime">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
position: relative;
left: -1px;
top: -2px;
border-left: none;
border-bottom: none;
border-top: none;
background:white;</string>
    </property>
    <property name="text">
     <string>E.T.:  00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="shocks">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
position: relative;
left: -1px;
border-left: none;
border-bottom: none;
background:white;</string>
    </property>
    <property name="text">
     <string>Shock:  0</string>
    </property>
   </widget>
   <widget class="QLabel" name="depthdisplay">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
position: relative;
left: -1px;
top: -1px;
border-left: none;
border-top: none;
background: white;</string>
    </property>
    <property name="text">
     <string>Dep: 0cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
